srcpkgs/xfce4-calculator-plugin/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'xfce4-calculator-plugin'
+pkgname=xfce4-calculator-plugin
+version=0.7.1
+revision=1
+build_style=gnu-configure
+configure_args="--with-locales-dir=/usr/share/locale"
+hostmakedepends="pkg-config intltool"
+makedepends="xfce4-panel-devel"
+short_desc="Calculator plugin for the Xfce4 panel"
+license="GPL-2.0-or-later, LGPL-2.0-or-later"
+homepage="https://docs.xfce.org/panel-plugins/xfce4-calculator-plugin"
+changelog="https://gitlab.xfce.org/panel-plugins/xfce4-calculator-plugin/-/blob/master/NEWS"
```

Comment:
link to a raw text file

Reply:
and so it goes. anyway, when xfce changes the server name, with this speed of maintainers and this approach, the packages will not be available for a long time. then they'll fix it.

Reply:
raw is prefered but not required

Reply:
> and so it goes. anyway, when xfce changes the server name, with this speed of maintainers and this approach, the packages will not be available for a long time. then they'll fix it.

I have no idea what you are talking about.

raw changelog is prefered which means packages should use it if it's available, please change it. Also, you will have to fix CI before this is merged.